src/Doctrine/JsonHelper.php

```diff
@@ -23,9 +23,19 @@ class JsonHelper
     public static function wrapJsonFunction(?string $where = null, ?string $slug = null, Connection $connection)
     {
         $version = new Version($connection);
+        //print_r($version->getPlatform()['driver_name']); # pgsql
+        //exit(print($where)); // translations_anyField.value
```

Comment:
Let's remove commented-out code

Reply:
Oh, sidenote: instead of `print_r`, use `dump()`. It's so much nicer. ;-) 

https://symfony.com/doc/current/components/var_dumper.html#dump-examples-and-output